LICENSE.txt

```diff
@@ -0,0 +1,11 @@
+Servus - A small set of network oriented utilities in C++ including a zeroconf implementation.
+
+Servus is licensed under the LGPL version 3, unless noted otherwise, e.g., for external dependencies. See file COPYING.txt for the full license. External dependencies are either LGPL or BSD-licensed. See file ACKNOWLEDGEMENTS.txt and AUTHORS.txt for further details.
```

Comment:
Lunchbox is LGPL V2, so this is relicensing. From a quick check, only @biddisco and @delyas are external and need to confirm. Guys - are you fine with this?